Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,97 @@
+const toDoInput = document.getElementById("inputToDo");
+const addToDoBtn = document.getElementById("addToDo");
+const container = document.getElementById("todo-list");
+const removeAll = document.getElementById("remove-all-completed");
```

Comment:
Why not name the identifies that identify button elements in your app consistently?

Reply:
yes, I updated the names for consistency.
Thanks CJ